src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -52,7 +52,7 @@ func AddKafkaFlags(
 		"config",
 		"",
 		"Redpanda config file, if not set the file will be searched for"+
-			" in the default locations",
+			" in /etc/redpanda/redpanda.yaml",
```

Comment:
+1 to improving this, but also we search, in order:
* cwd
* home (which I'm kinda ok with deprecating -- searching $HOME is weird)
* lastly, /etc/redpanda/redpanda.yaml
so can this change to `$PWD or /etc/redpanda/redpanda.yaml`?

Reply:
Good point, fixed. Thanks